lib/utils/errorHandling/ResponseErrorContainer.test.js

```diff
@@ -0,0 +1,66 @@
+/* Developed collaboratively using AI (GitHub Copilot) */
```

Comment:
we used in BE similar appraoch, it good, will gather statistic of usage based on this
https://github.com/search?q=repo%3Afolio-org%2Fmod-orders%20copilotgen&type=code